config/dev/openstack-credentials.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Forgot to ask: do we need a doc on how to setup all of the envs here? I might have missed it in other PRs